docs/src/components/Layout/GlobalNav/GlobalNav.module.scss

```diff
@@ -0,0 +1,174 @@
+// This class is added to the sites that aren't using the Amplify UI theme provider and is creating the variables that are being used in dark mode of Amplify UI
+.add-variables {
+  --amplify-colors-font-primary: #232f3e;
+  --amplify-colors-font-tertiary: #545b64;
+  --amplify-colors-background-primary: white;
+  --amplify-colors-font-hover: #000;
+}
+
+.navbar {
+  box-shadow: 0px 4px 16px 0px rgba(0, 0, 0, 0.04);
+  border-bottom: 1px solid var(--navbar-border-color);
+  background-color: var(--amplify-colors-background-primary);
+  position: sticky;
+  top: 0px;
+  z-index: 200;
+
+  a,
+  p,
+  span {
+    font-family: 'Amazon Ember' !important;
+    -webkit-font-smoothing: auto !important;
+    -moz-osx-font-smoothing: auto !important;
+  }
+
+  // Top level navbar element, create all of the variables that will be used throughout the unified navbar
+  --navbar-border-color: #d5dbdb;
+  --navbar-current-property-color: #ff9900;
+  --secondary-nav-hover-color: black;
+  --secondary-nav-hover-background-color: white;
+  --dev-center-nav-height: 60px;
+  --amplify-colors-font-hover: #000;
+}
+
+[data-amplify-theme='amplify-docs'][data-amplify-color-mode='dark'] .navbar {
+  --amplify-colors-font-hover: inherit;
+}
+
+[data-amplify-theme='amplify-docs'][data-amplify-color-mode='system'] .navbar {
+  @media (prefers-color-scheme: dark) {
+    --amplify-colors-font-hover: inherit;
+  }
+}
+
+.dev-center-logo {
+  color: var(--amplify-colors-font-primary);
+  font-size: 1.25rem;
+  white-space: nowrap;
+}
+
+.nav-container {
+  display: flex;
+  height: var(--dev-center-nav-height);
+  align-items: center;
+  justify-content: space-between;
+}
+
+.mobile-nav-container {
+  display: flex;
+  height: var(--dev-center-nav-height);
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav-links-container {
+  height: var(--dev-center-nav-height);
+  align-items: center;
+  justify-content: space-between;
+}
+
+.mobile-nav-menu-container {
+  z-index: 1000;
+  background-color: var(--amplify-colors-background-primary);
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-bottom: 1px solid var(--navbar-border-color);
+}
+
+.mobile-nav-icons-container {
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+
+  svg {
+    display: block;
+    margin: auto;
+  }
+}
+
+.mobile-nav-menu-items {
+  border-top: 1px solid var(--navbar-border-color);
+  height: 48px;
+  width: 100%;
+}
+
+.nav-menu-item {
+  padding: 12px;
+  display: flex;
+  align-items: center;
+  height: 100%;
+  font-size: 1rem;
+  text-decoration: none;
+  color: var(--amplify-colors-font-tertiary) !important;
+  font-family: 'Amazon Ember';
+  cursor: pointer;
+  font-weight: 400;
+
+  white-space: nowrap;
+
+  &:hover,
+  &:focus {
+    color: var(--amplify-colors-font-hover) !important;
```

Comment:
We have the `!important` rules are here because we're using this class to override the `amplify-link` class since this class is applied to the `Link` component.

We could probably move the `color` CSS directly to the `Link` element if you want, but there isn't a way to do that with the `:hover` and `:focus` is there?